cmd/juju/application/deploy.go

```diff
@@ -444,7 +448,7 @@ var (
 	// charmOnlyFlags and bundleOnlyFlags are used to validate flags based on
 	// whether we are deploying a charm or a bundle.
 	charmOnlyFlags  = []string{"bind", "config", "constraints", "force", "n", "num-units", "series", "to", "resource"}
-	bundleOnlyFlags = []string{}
```

Comment:
Lucky this already existed! Was it vestigial?

Reply:
I don't know.